### PR TITLE
[Chore] Update README.md

### PR DESCRIPTION
**Pull request by chiiida**
## What happened 👀

- Add a short introduction and Getting Started to `README.md`
 
## Proof Of Work 📹

`N/A`

### CODE REVIEW

**Review by blyscuit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
-# Git Repository Template
+# Github Notifications Application
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+## What is Github Notifications Application?
+Github Notifications is a macOS application to manage Github Notifications to be more personalized. Receive notifications on your Mac when there are new updates on subscribed repositories.
```
Comment (vnntsu) [inline README.md:4]:
```suggestion
Github Notification Application is a macOS application for managing and personalized Github's notifications.
```

Comment (vnntsu) [reply README.md:4]: In the second sentence, I suggest you list down the actions that trigger to show notification, f.e: mention, request review, etc.

**Review by vnntsu — APPROVED** [review]

**Review by suho — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
-# Git Repository Template
+# Github Notifications Application
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+## What is Github Notifications Application?
+Github Notification Application is a macOS application for managing and personalized Github's notifications. Receive notifications from subscribed repositories on your Mac when you were assigned, mentioned, review requested, and when your pull request was reviewed.
```
Comment (markgravity) [inline README.md:4]:
```
Github Notification Application is a macOS application for managing and personalized Github's notifications. 
It receives notifications from subscribed repositories on your Mac when you were assigned, mentioned, review requested, and when your pull request was reviewed.
```
What do you think? 🤔

**Review by markgravity — APPROVED** [review]
LGTM

**Pull request merged**